### ISSUE TITLE
Which Nuget Package?

### ISSUE THREAD

**Issue by DoctorEclipse** [opentk/opentk#400]
I've been following the recent development of OpenTK and I saw that the old nuget package was abandoned since no one could get a hold of the owner. I was going to change to OpenTK.Next today when my nuget manager told me that the old package was updated. I checked out the nuget page for it and apparently the owner released a [new version](https://www.nuget.org/packages/OpenTK/) yesterday based on this repositorey. No one has downloaded it yet. Would it be better to use that instead? I always thought OpenTK.Next was going to be for glNext/Vulkan. I apologize if this was already covered and I missed something.

**Comment by varon** [comment]
@DoctorEclipse Thanks for the really good question!

At the moment, we've got no communication with @BenjaminNitschke, aka @exdreamduck, despite numerous emails and mentions. We have no idea what he's doing with the package, which he currently has sole control over. It's why we've named the current pre-release package OpenTK.Next.

In all honestly, I would advise you stick with the pre-release package, which is built against dev branch of OpenTK itself. This is representative of the project's direction, and built by the maintainers of this repo.

After we've decided on the exact direction we want to go with OpenTK, we'll probably end up trying to reclaim the package ID. It's our hope that we can do this by co-operation with @BenjaminNitschke without relying on the somewhat heavy-handed NuGet abuse report system.

TL;DR - For now, OpenTK.Next pre-release is the package to use.

**Comment by DoctorEclipse** [comment]
It's definitely strange that he would release an update and keep everyone else in the dark. Well thank you for responding so quickly and of course for helping to continue the project. This can be closed now unless anyone has anything else to add.
